en/concept/flight_modes.md

```diff
@@ -1,34 +1,36 @@
 # Flight Modes
 
-**Flight Modes** define the state of the system at any given time.  The user transitions between flight modes via switches on the remote control or the [ground control station](../qgc/README.md).
+**Flight Modes** define the state of the system at any given time.  The user transitions between flight modes via switches on the remote control or the [ground control station](../qgc/README.md), and are loosely grouped according to their level of control of the vehicle by the system vs. the user: "manual" modes, where the controller is essentially a pass-through from the user's remote control to the vehicle's actuators; "assisted" modes, where the system's outputs are coordinated with the user's commands; and, "auto" modes, where the controller requires little to no user input whatsoever.
```

Comment:
I like the description, but I'd like to suggest improvements on the wording (feel free to not consider them).

> **Flight Modes** define the state of the system at any given time.  The user transitions between flight modes via switches on the remote control or using the [ground control station](../qgc/README.md). They are loosely grouped according to the level of control of the vehicle by the system (respectively the user):
> * "Manual" modes are those where the user has the most control over the vehicle. A slightly simplified way to see it is as if the controller was essentially a pass-through from the user's remote control to the vehicle's actuators.
> * "Assisted" modes are those where the system's outputs are coordinated with the user's commands.
> * "Auto" modes are those where the controller requires little to no user input whatsoever.